keps/sig-network/1669-graceful-termination-local-external-traffic-policy/README.md

```diff
@@ -141,6 +149,266 @@ This would either happen if a version of the control plane was not aware of the
 
 There's not much risk involved as the worse case scenario is falling back to existing behavior.
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [X] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: ProxyTerminatingEndpoints
+  - Components depending on the feature gate: kube-proxy
+
+###### Does enabling the feature change any default behavior?
+
+Yes, when externalTrafficPolicy=Local and there are only terimating endpoints,
+kube-proxy will route traffic to those endpoints. Before this change, kube-proxy
+dropped this traffic instead.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+kube-proxy will drop traffic if only terminating endpoints are available.
```

Comment:
will no longer drop traffic right?

[i.e. it will drop traffic after dsiablement, but after reenablement it will stop dropping traffic again]

Reply:
ah yes, thanks for catching that